src/js/media-theme-element.js

```diff
@@ -1,4 +1,4 @@
-import { window } from './utils/server-safe-globals.js';
+import { window, document } from './utils/server-safe-globals.js';
```

Comment:
NOTE: This is unrelated but got caught while testing. There's a module-level reference to `document` that was causing crashes for SSR usage.